src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1075,6 +1032,116 @@ else if (fieldClass.isEnum()) {
         return result;
     }
 
+    /**
+     * Checks whether property is a dynamic custom expression.
+     *
+     * @param sectionName The name of the section/module being worked on.
+     * @param propertyName The property name to work with.
+     * @return true if it is a dynamic custom expression.
+     */
+    private static boolean isDynamicCustomExpression(String sectionName, String propertyName) {
+        return ("SuppressionCommentFilter".equals(sectionName)
+                || "SuppressWithNearbyCommentFilter".equals(sectionName)
+                || "SuppressWithPlainTextCommentFilter".equals(sectionName))
+                && ("checkFormat".equals(propertyName)
+                || "messageFormat".equals(propertyName)
+                || "idFormat".equals(propertyName)
+                || "influenceFormat".equals(propertyName))
+                || ("RegexpMultiline".equals(sectionName)
+                || "RegexpSingleline".equals(sectionName)
+                || "RegexpSinglelineJava".equals(sectionName))
+                && "format".equals(propertyName);
+    }
+
+    /**
+     * Get's the name of the bean property's type for the Int[] class.
+     *
+     * @param sectionName The name of the section/module being worked on.
```

Comment:
Please use lowercase "the" for `@param` section. The Javadoc looks bad:
![image](https://user-images.githubusercontent.com/900805/127449089-581606be-c3d8-48d6-b8a5-b948343618b9.png)

Reply:
Done